Source/WebCore/layout/formattingContexts/inline/InlineLineBuilder.h

```diff
@@ -103,6 +103,7 @@ class LineBuilder {
         std::optional<InlineLayoutUnit> hintForNextLineTopToAvoidIntrusiveFloat { }; // This is only used for cases when intrusive floats prevent any content placement at current vertical position.
     };
     LayoutResult layoutInlineContent(const LineInput&, const std::optional<PreviousLine>&);
+    Vector<size_t> listBreakOpportunities(const InlineItemRange& layoutRange);
```

Comment:
does this have to be on the LineBuilder? LineBuilder's only task is to layout_inline_content.